release-notes.txt

```diff
@@ -1,3 +1,548 @@
+* mbox: provide function to unset initialized mbox (#15485)
+
+System Libraries
+----------------
+
++ add error check of aes_encrypt() (#15576)
++ event_periodic_callback: add event_periodic_callback_create() (#20459)
++ examples/rust-gcoap: Add SAUL (#17554)
++ shell/cmds: add udptty command (#19742)
++ sys/libc: add endian.h (#20310)
++ sys/shell/lwip: add IPv4 configuration to lwip ifconfig command (#19849)
++ tests: Add NULL pointer argument test for fmt_lpad/to_lower/str (#20344)
+* makefiles/cargo: Remove CARGO_CHANNEL and other simplifications (#20319)
+* Monitoring malloc usage (#20363)
+* rust: Make dependencies visible (#18684)
+* sys/checksum: Remove deprecated crc16_ccitt functions (#20527)
+* sys/hashes: support for SHA-{384,512/{224,256}} (#20484)
+* sys/isrpipe/read_timeout: don't set up timer if data is available (#20558)
+* sys/newlib_syscalls_default: fix race condition in __sinit() (#20392)
+* sys/psa_crypto: use SHA256 CSPRNG as default (#20460)
+* sys/shell/doc: Shell access is root access (#20546)
+* sys/suit: Remove deprecated suit_coap_run function (#20528)
+* sys/suit: return error when URL buffer is too small (#20559)
+* sys/tiny_strerror: fix compilation on LLVM (#20483)
+
+Networking
+----------
+
++ gcoap: add some client-side observe handling (#20073)
++ nanocoap: add coap_build_reply_header() (#20284)
++ nanocoap: introduce coap_get_method() (#20191)
++ nanocoap_sock: add support for IPv4 (#20476)
++ net/sock_dtls: add public key verification (#20048)
++ pkg/wakaama: Notify resource updates when writing to resources (#20326)
++ sys/net/ipv4: add IPV4_ADDR_INIT() macro (#20354)
+* always use nanocoap_resources instead of manually assembling
+  coap_resources[] (#20374)
+* cord_lc: Process truncated reads (#20547)
+* gnrc_ipv6_nib: Ignore PIO with on-link flag (#20418)
+* nanocoap/cache: Extend with option-only cache keygen (#20043)
+* pkg/tinydtls: enforce the default dtls user params to be configurable (#20478)
+* routing/rpl: Fixing styling & format (#20294)
+* sys/net/gnrc: Remove code duplication (#16502)
+
+Packages
+--------
+
++ pkg/wakaama: add Light Control object implementation (#16204)
+* ci: support git-cache-rs (#20311)
+* Makefile.{include,base},pkg/nanopb: fix target deps & code generation (#20400)
+* pkg/flashdb: enhance FAL config (#20221)
+* pkg/uzlib: warn about memory leak during compression (#20293)
+
+Boards
+------
+
++ boards/cpu: Add GBA boot support to RIOT (#19519)
++ boards/nucleo-c031c6: add PWM configuration (#20520)
++ boards/nucleo-u575zi-q: Add support (#20444)
++ boards/weact-g030f6: add support for minimal break-out board (#20337)
++ cpu/rpx0xx: add initial pwm support (#20473)
++ cpu/stm32: add support for STM32C0 and NUCLEO-C031C6 (#20300)
+* boards/feather-nrf52840-sense: declare lsm6ds33 accelerometer (#20436)
+* boards/feather-nrf52840-sense: use generic lsm6dsxx (#20526)
+* cpu/msp430/f2xx: clean up periph_uart,periph_spi (#20357)
+* cpu/native: Enable Rust on 64bit (#20339)
```

Comment:
```suggestion
```

As that feature is introduced in this release, adding Rust support does not deserve its own mention.